Huffman/Huffman/Huffman.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using HTree;
+
+namespace HuffmanCompressor;
+
+public class Huffman
+{
+    public static void Compress(string filename)
+    {
+        byte[] fileData = File.ReadAllBytes(filename);
+
+        // Counting byte frequency
+        Dictionary<byte, int> frequencies = new Dictionary<byte, int>();
```

Comment:
```suggestion
        Dictionary<byte, int> frequencies = new();
```
Не пишите один и тот же тип дважды без нужды.